Makefile

```diff
@@ -127,6 +128,9 @@ $(CONVERSION_GEN): $(TOOLS_DIR)/go.mod
 $(RELEASE_NOTES) : $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR) && go build -tags tools -o $(BIN_DIR)/release-notes sigs.k8s.io/cluster-api/hack/tools/release
 
+$(TEMPLIFY) : $(TOOLS_DIR)/go.mod
+	cd $(TOOLS_DIR) && go build -tags tools -o $(BIN_DIR)/templify github.com/wlbr/templify
```

Comment:
I read your note about go.rice in the other thread, although this particular project seems very new to take a dependency on. Can we try go-bindata, which we already use upstream?

@fabriziopandini maybe you can help us here to figure out what's going on with go.rice and GOPATH?

Reply:
go-bindata is dead.

In any case, I don't really want the rigmarole of an entire VFS for a single embedded string (makes sense for clusterctl, not so much here). I'm just going to manually constantise it.

Reply:
go-bindata is dead.

In any case, I don't really want the rigmarole of an entire VFS for a single embedded string (makes sense for clusterctl, not so much here). I'm just going to manually constantise it.

Reply:
go-bindata is dead.

In any case, I don't really want the rigmarole of an entire VFS for a single embedded string (makes sense for clusterctl, not so much here). I'm just going to manually constantise it.